### PR TITLE
SONAR-7867 persist date of project's snapshot as created_at of ProjectView's snapshot

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/component/ProjectViewAttributes.java [hunk 1]
```diff
@@ -19,24 +19,43 @@
...
+  public ProjectViewAttributes(long projectId, @Nullable Long analysisDate) {
     this.projectId = projectId;
+    this.analysisDate = analysisDate;
   }
 
   public long getProjectId() {
     return projectId;
   }
 
+  public Long getAnalysisDate() {
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/computation/component/ProjectViewAttributes.java]: missing @CheckForNull (or return an Optional)

File: server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistSnapshotsStep.java [hunk 2]
```diff
@@ -185,6 +185,16 @@ private SnapshotDto createSnapshot(Component component, Path<SnapshotDtoHolder>
       return snapshotDto;
     }
 
+    private Long computeSnapshotAnalysisDate(Component component, long analysisDate) {
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistSnapshotsStep.java]: can return primitive type `long`

**Comment by simonbrandhof** [comment]
LGTM

File: server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistSnapshotsStep.java (hunk 2 above)
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/step/PersistSnapshotsStep.java]: true

File: server/sonar-server/src/main/java/org/sonar/server/computation/component/ProjectViewAttributes.java (hunk 1 above)
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/component/ProjectViewAttributes.java]: indeed

**Pull request merged**